scopes/component/component/component.graphql.ts

```diff
@@ -157,6 +165,18 @@ export function componentSchema(componentExtension: ComponentMain) {
         aspects: (component: Component, { include }: { include?: string[] }) => {
           return component.state.aspects.filter(include).serialize();
         },
+        snaps: async (component: Component, { snapFilter }: { snapFilter?: { type?: string; take?: number } }) => {
```

Comment:
https://github.com/teambit/bit/blob/803b02d8f17b7ea66a3b90320d455e73598e52e8/scopes/component/component/component.graphql.ts#L171
isn't this enough for you?

Reply:
https://github.com/teambit/bit/pull/5444#discussion_r815437086

With the addition of the new version dropdown,
Component UI will show all the `versions`, `snaps`, and `lanes`
To accommodate that, I used the query you referenced above in Menu.tsx (components/ui)
But, @KutnerUri recommended to update the component graphql to return the snaps data instead of making an additional query

Reply:
ah, I didn't know we have this api.
Though technically we could use it in the same request, shouldn't it be part of the component API? @GiladShoham